tests/python/pants_test/base/test_exception_sink_integration.py

```diff
@@ -108,13 +107,12 @@ def _send_signal_to_waiter_handle(self, signum):
       # Return the (failed) pants execution result.
       yield (workdir, waiter_run)
 
-  @unittest.skip('Flaky: https://github.com/pantsbuild/pants/issues/6708')
   def test_dumps_logs_on_terminate(self):
     # Send a SIGTERM to the local pants process.
     with self._send_signal_to_waiter_handle(signal.SIGTERM) as (workdir, waiter_run):
       assertRegex(self, waiter_run.stderr_data, """\
 timestamp: ([^\n]+)
-Signal {signum} was raised. Exiting with failure. \\(backtrace omitted\\)
+Signal {signum} (SIGTERM) was raised. Exiting with failure. \\(backtrace omitted\\)
```

Comment:
try with \(SIGTERM\)

Reply:
Spent far too long figuring this one out! Thank you!